stdlib/src/builtin/dtype.mojo

```diff
@@ -727,7 +727,7 @@ fn _get_runtime_dtype_size(type: DType) -> Int:
     statically known dtypes. Instead, we have to perform a dispatch to
     determine the size of the dtype.
     """
-    alias type_list = List[DType](
+    alias type_list = StaticTuple[DType, 16](
```

Comment:
**Suggestion** Use `InlineArray` instead.  We're about to mark `StaticTuple` as deprecated and soon remove it.

Reply:
InlineArray gets the error: "cannot use a dynamic value in alias initializer".  From the line `alias concrete_type = type_list[idx]`.  This is the same error `List` also gets with the nightly 2024.5.1002.  I assume because `InlineArray` and `List` make very similar use of `__refitem__`.